Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -2448,6 +2462,42 @@ static NSStringCompareOptions compareOptionsForFindOptions(WebCore::FindOptions
     });
 }
 
+RefPtr<TextIndicator> UnifiedPDFPlugin::textIndicatorForSelection(OptionSet<WebCore::TextIndicatorOption> options, WebCore::TextIndicatorPresentationTransition transition)
+{
+    if (auto selectionBounds = selectionBoundsForFirstPageInDocumentSpace(m_currentSelection)) {
```

Comment:
Would be better to early return so the rest of the function is not indented.

Reply:
I'm confused by "selectionBoundsForFirstPageInDocumentSpace". First page of the document, or first page of the selection? Why is it OK to only consider the first page?

Reply:
Find highlights spanning multiple pages will just highlight the beginning of the match, like their Lookup and Define siblings. Room for improvement.